test/mapping/attractor_mapping.jl

```diff
@@ -218,9 +245,9 @@ if DO_EXTENSIVE_TESTS
             g = exp(entropy(Renyi(; q = 0), probs))
             return SVector(g, minimum(A[:,1]))
         end
-
+        
         test_basins(ds, u0s, grid, expected_fs_raw, featurizer;
-        ε = 0.01, ferr=1e-2, Δt = 0.2, mx_chk_att = 20)
+        ε = 0.01, ferr=1e-2, Δt = 0.2, mx_chk_att = 20, distance_threshold_pairwise=100) #threshold is very high because features haven't really converged yet here
```

Comment:
How can the threshold be larger than 1? If we scale to 0-1 by default, the distance in feature space can never be more than the sqrt of the number of feature dimensions.

Reply:
Ah. we dont rescale be default. I think we should.

Reply:
I don't think we should, there are some serious problems with rescaling the features, and the user might get lost debugging the algorithm until they realize what the rescaling did with the features. We discussed this a bit in PR #59 but actually there's more to it I think. I will write my ideas about this down soon and then we can see.

For now I defend that rescaling should be done with a lot of care, so it should be disabled by default.